crAPI/deploy/k8s/gcp/web/ingress.yaml

```diff
@@ -15,4 +15,4 @@ spec:
   type: LoadBalancer
   selector:
     app: crapi-web
-  loadBalancerIP: "35.226.144.72"
+  # loadBalancerIP: "35.226.144.72"
```

Comment:
Is this an intentional change?

Reply:
Yes, since we're hosted in Azure, we only need to pass an IP if we're assigning it a static, pre-allocated IP.
It's only being done manually for now.
I'll add the different IPs for crapi, crapi-dev and prod-demo to a cron workflow later.